### ISSUE TITLE
[Feature]: Uploaded video files of format (.mov, .avi) don'nt play 

### ISSUE THREAD

**Issue by Himanshu664** [RocketChat/Rocket.Chat#24263]
### Description:

Uploaded video files of format (.mov, .avi) don'nt play in the player, needed support for the .mov, .avi format.
As support for .mp4 and .webm is there already.

### Steps to reproduce:

1. Go to messages(group,dm)
2. Click on upload files
3. Upload any video of format( .mov, .avi)

### Expected behavior:
Video of format .mov , .avi does'nt playing in player.

https://user-images.githubusercontent.com/76220055/150678320-1d8d229b-d33b-465a-a06e-5b4f6bc51684.mp4

### Actual behavior:

Video of .mov , .avi format should play in video player 

### Server Setup Information:

Version of Rocket.Chat Server: 4.4.0-develop
Operating System: Ubuntu 20.04

### Client Setup Information

Desktop App or Browser Version: Browser
Operating System: Linux

**Comment by Ayush01729** [comment]
Hello @Himanshu664 , I am working on this issue.

**Comment by MURTOMAASORTAJA** [comment]
Yup. Can confirm. Uploading a video from iPhone just arrives as an audio, if anything.

**Comment by IVIJL** [comment]
This is still issue in 7/23 :(

**Comment by elDaniello** [comment]
Still not working

**Comment by MURTOMAASORTAJA** [comment]
> This is still issue in 7/23 :(

> Still not working

Yeah. I guess it's time to move on. 😄
Mattermost works well!
